kbase-extension/static/kbase/js/common/spec.js

```diff
@@ -71,10 +71,34 @@ define(['bluebird', 'common/lang', 'common/sdk', 'widgets/appWidgets2/validators
             return Promise.props(validationMap);
         }
 
+        /**
```

Comment:
This is here because we need to iterate over a list of a subset of parameters with different values. The configure tab breaks the parameter list into two segments, and we need to address them separately.